config/global.json

```diff
@@ -77,6 +78,10 @@
                 "host": false,
                 "port": false
             }
+        },
+        "exitCode": {
```

Comment:
Do not add this to configuration files. Define it in file where it's used. It's not sth that should be configurable. It's a standard.